File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/non-plagiarized/06/T05.kt
package IR

import java.util.Scanner

/**
 *
 * @author 92E0988C1682C76D4D307AA15EC8346E
 */
fun main() {
    val sc = Scanner(System.`in`)
    print("Enter an integer : ")
    val number = sc.nextInt()
    reverse(number)
}

fun reverse(number: Int) {
    val numberString = number.toString()
    for (i in numberString.length - 1 downTo 0) {
        print(numberString[i])
    }
    println("")
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/06/T05.py
import sys

def reverse(number):
    number_string = str(number)
    for i in range(len(number_string), 0, -1):
        print(number_string[i-1], end='')
    print()

if __name__ == "__main__":
    number = int(input("Enter an integer: "))
    reverse(number)

